web/app/themes/wordpress-scaffold/lib/Grrr/Utils/Security.php

```diff
@@ -0,0 +1,23 @@
+<?php namespace Grrr\Utils;
+
+/**
+ * Disable REST API link in HTTP headers, such as:
+ * `Link: <https://example.com/wp-json/>; rel="https://api.w.org/"`
+ * Taken from: https://wordpress.org/plugins/disable-wp-rest-api/
+ */
+remove_action('template_redirect', 'rest_output_link_header', 11);
+
+/**
+ * Disable REST API for unauthenticated users.
+ * Taken from: https://wordpress.org/plugins/disable-wp-rest-api/
+ */
+function disable_wp_rest_api($access) {
+    if (!is_user_logged_in()) {
+        $message = 'REST API restricted to authenticated users.';
+        return new \WP_Error('rest_login_required', $message, [
+            'status' => rest_authorization_required_code(),
+        ]);
+    }
+    return $access;
+}
+add_filter('rest_authentication_errors', __NAMESPACE__ . '\\disable_wp_rest_api');
```

Comment:
Dit willen we eigenlijk alleen als we de static site generator opzet gebruiken. Want de rest api voor formulieren moeten wel gebruikt kunnen worden als guest.

Reply:
As discussed in person, we want to keep everything closed. But this method didn't provide a simple way to enable certain endpoints. Changes coming up 👍